scalyr_agent/copying_manager.py

```diff
@@ -289,12 +299,16 @@ def add_log_config( self, monitor, log_config ):
 
         self.__lock.acquire()
         try:
-            if log_config['path'] not in self.__all_paths:
+            path = self.__path_in_all_paths( log_config['path'], self.__all_paths )
+            if path is None:
                 log.log(scalyr_logging.DEBUG_LEVEL_0, 'Adding new log file \'%s\' for monitor \'%s\'' % (log_config['path'], monitor.module_name ) )
                 self.__pending_log_matchers.append( LogMatcher( self.__config, log_config ) )
                 self.__all_paths[log_config['path']] = 1
+                path = log_config['path']
             else:
-                self.__all_paths[log_config['path']] += 1
+                log.log(scalyr_logging.DEBUG_LEVEL_0, 'New log file \'%s\' already matches \'%s\' for monitor \'%s\'' % (log_config['path'], path, monitor.module_name ) )
+                if path == log_config['path']:
```

Comment:
Add comment as to why you check whether or not `path` is equal to `log_config['path']`

Reply:
Done.